### PR TITLE
Lomd documentation update

### PR DESCRIPTION
**Pull request by siliu-tacc**
Fix typos and bugs in Lmod documentation

### CODE REVIEW

File: docs/source/010_user.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ User Guide for Lmod
 The guide here explains how to use modules. The User's tour of the
 module command covers the basic uses of modules. The other sections
 can be read at a later date as issues come up. The Advance user's
```
Comment (justbennet) [inline docs/source/010_user.rst]: This should be Advanced User's

Comment (justbennet) [inline docs/source/010_user.rst]: I think this should be Guide, not guild

File: docs/source/010_user.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@ User Guide for Lmod
...
-guilde is for users needing to create their own modulefiles.
+guild is for users needing to create their own modulefiles.
```
Comment (justbennet) [inline docs/source/010_user.rst]: This should be Guide, not guild, I think, as it continues "Advanced User's Guide" from the line above.

**Comment by rtmclay** [comment]
Thanks to both of you.  I have finally merged in this pull request.

**Pull request merged**